Tasks/AzureFunctionAppV1/taskparameters.ts

```diff
@@ -28,12 +29,17 @@ export class TaskParametersUtility {
         }  
 
         taskParameters.azureEndpoint = await new AzureRMEndpoint(taskParameters.connectedServiceName).getEndpoint();
-        console.log(tl.loc('GotconnectiondetailsforazureRMWebApp0', taskParameters.WebAppName));   
+        console.log(tl.loc('GotconnectiondetailsforazureRMWebApp0', taskParameters.WebAppName));
 
         var appDetails = await this.getWebAppKind(taskParameters);
         taskParameters.ResourceGroupName = appDetails["resourceGroupName"];
         taskParameters.WebAppKind = appDetails["webAppKind"];
+        taskParameters.sku = appDetails["sku"];
 
+        if(taskParameters.sku.toLowerCase() == skuDynamicValue) {
+            taskParameters.isConsumption = true;
```

Comment:
Can yoiu directky set as taskParameters.isConsumption = taskParameters.sku.toLowerCase() == skuDynamicValue
This will ensure value is always set as true/false, otherwise when not true it will be null and can give null exception later.

Reply:
It is done.